Dockerfile

```diff
@@ -85,10 +88,11 @@ RUN mkdir -p rootfs-overlay && \
 
 COPY package.json package-lock.json /usr/src/app/
 
+# install the newer NPM so we can use npm ci
+RUN npm install -g npm@6
```

Comment:
You should be able to move this `RUN npm install -g npm@6` and the `RUN apt-get update ...` above to another build step and share the layers between `node-deps` and `node-build` in order to speed up builds/increase caching